docs/Installing-Lmod-without-root-permissions.rst

```diff
@@ -67,18 +75,25 @@ Dependencies: building Lmod using the steps below requires tcl, tcl-dev(el), mak
 
     ./configure --prefix=$HOME && make install
 
-**Step 3**: Update ``$PATH`` so ``lmod`` is available (put this in your
-``.bashrc``):
+**Step 3**: Update the ``$LMOD_CMD`` to point to the Lmod executable
+(put this in your ``.bashrc``):
+
+.. code:: bash
+    export LMOD_CMD=$HOME/lmod/8.4/libexec/lmod
```

Comment:
This is set by the Lmod init script, no need to do this manually at all (and if `$LMOD_CMD` is set, no need to fiddle with `$PATH`, indeed)